src/components/LoginPanel.tsx

```diff
@@ -31,6 +38,10 @@ const MiddleDiv = styled.div`
         -webkit-box-shadow: 0 0 0 9999px ${(props) => props.theme.main.background[2]} inset !important;
         -webkit-text-fill-color: ${(props) => props.theme.main.foreground[2]} !important;
     }
+
+    @media (min-width: 500px) {
+        padding: 30px 80px;
+    }
```

Comment:
This doesn't quite look correct on some small viewports (as per our conversation), could you look into it?